chroma-agent/tests/test_node_admin.py

```diff
@@ -38,11 +23,25 @@ def test_unmanage_network_nm_running(self):
         self.assertRanAllCommandsInOrder()
         self.write_ifcfg_mock.assert_called_with(self.device_name, self.mac_address, None, None)
 
-    def test_unmanage_network_nm_failures(self):
-        for expected_rc in [2, 127, 8]:
+    def test_unmanage_network_nm_not_installed(self):
+        with mock.patch('chroma_agent.lib.shell.AgentShell.try_run', side_effect=OSError):
+            self.assertIsNone(node_admin.unmanage_network(self.device_name, self.mac_address))
+
+    def test_unmanage_network_nm_known_failures(self):
+        for expected_rc in [8]:
```

Comment:
If we used a constant instead of 8 I think the test would read a little cleaner.